### ISSUE TITLE
When breaking a slugma burner by hand it should release the slugma as a wild Pokémon with all the saved stats.

### ISSUE THREAD

**Issue by ghost** [R3charged/Create-Slugma#11]
I thought of this feature when I broke the restoration tank by hand and the Pokémon inside of it came out as a wild Pokémon and thought that it could be a good feature if Slugma burners didn't just delete Slugma out of the existance when the burner was broken but instead came out as a wild Pokémon.

**Comment by R3charged** [comment]
Good idea, and if player owns a slugma in the cage, it should return to that player's box
